src/ic-cdk/src/api/wasi.rs

```diff
@@ -0,0 +1,26 @@
+type Fd = u32;
```

Comment:
Could we say something here at a really high level about why this exists?  Is it so canisters using the cdk can call `print!()` and `println!()`?

Also, is there anything to mention in the changelog about this?

Reply:
This module is considered to be a implementation detail of `export_candid` macro. The methods in this module should not be used in canister code.

I'll hide the whole module in doc and make it clearer in the comment.